### ISSUE TITLE
Market order

### ISSUE THREAD

**Issue by helger** [coinigy/api#7]
Does the Coinigy API support placing market orders?

**Comment by robborden** [comment]
Currently we do not accept market orders specifically.  However, a market order is simply a limit order set at a level which will instantly fill your entire order.  You could set a limit buy order at a price high above the last price, or a limit sell order at a price low enough below the last price that your order will be completely filled instantly at the best price possible.  That is the same thing a market order does.

**Comment by helger** [comment]
Ok, thanks. And the stop orders, are they always stop-limit orders?

**Comment by geekguy** [comment]
@robborden  - will all the exchange process such high/low limit orders at best possible rate?

**Comment by GalGreenfield** [comment]
@robborden Do you have plans to add that to the API anytime soon?
Some exchanges already support market orders.
Even partial support of market orders in terms of supporting market orders for some of the exchanges that you enable trading with via the API would be useful for people who use your API to make orders.

**Comment by jdat82** [comment]
Yes, plus a MARKET order might be very useful in combination with a STOP orders.
